app/web_framework/cluster_master_application.py

```diff
@@ -60,11 +60,27 @@ def initialize(self, route_node=None, cluster_master=None):
         super().initialize(route_node)
 
 
-class _RootHandler(_ClusterMasterBaseHandler):
+class _ClusterMasterBaseStaticFileHandler(ClusterBaseHandler, tornado.web.StaticFileHandler):
```

Comment:
My original comment was maybe not very clear -- I was suggesting getting rid of `_ClusterMasterBaseStaticFileHandler` entirely and just making `_BuildResultHandler` do the multiple inheritance? What do you think of that?